lib/bankaccounts/account.rb

```diff
@@ -0,0 +1,53 @@
+module BankAccounts
+
+  class Account
+    attr_accessor :id, :min_balance, :initial_balance, :balance, :fee
```

Comment:
Oh, and given the context, `attr_accessor` gives both read and write access to the attribute. You may want to use `attr_reader` to users can't arbitrarily change the attributes from outside